### PR TITLE
Managed instance cost calculation guidance

### PR DESCRIPTION
**Pull request by dan-mckean**

### CODE REVIEW

File: handbook/engineering/distribution/managed/cost_estimation.md [hunk 1]
```diff
@@ -1,6 +1,52 @@
...
-The following describes how to **estimate** the infrastructure costs incurred for a managed instance. Managed instances are always created in completely isolated GCP projects, and as such it is easy to see the exact infrastructure cost breakdown for a given customer.
+## How to calculate the cost of a managed instance
+
+> ⚠️ **IMPORTANT:** This provides a _rough estimate_ to the best of our ability, infrastructure pricing estimates are not trivial to do accurately and actual costs are always subject to change. The goal of this is to give you a rough estimate of costs, not an exact amount.<br><br>ℹ️ If unsure about any anything with pricing for a managed instance, message `@distribution-support` in the [#distributioneers slack channel](https://sourcegraph.slack.com/archives/CJX299FGE) – ensure you include all the information needed for the resource calculator and any extra information about why you need help.
+
+Several data points are needed to work out the cost:
+- Size of all of their repositories on disk
+- Number of repositories
+- Number of those repositories that are monorepos
+
```
Comment (bobheadxi) [inline handbook/engineering/distribution/managed/cost_estimation.md:11]:
```suggestion
Several data points are needed to work out the cost:

- Size of all of their repositories on disk
- Number of repositories
- Number of those repositories that are monorepos

These data points are also listed in the CSE [managed instance request template](https://github.com/sourcegraph/customer/blob/master/.github/ISSUE_TEMPLATE/new_managed_instance.md).
```

File: handbook/engineering/distribution/managed/cost_estimation.md [hunk 2]
```diff
@@ -1,6 +1,52 @@
...
+
+| Cost estimate  | Description                                                                                                    |
+|----------------|----------------------------------------------------------------------------------------------------------------|
+| $282/mo        | Data and snapshot storage costs for the first 250GB                                                            |
+| +$1/GB storage | For any additional storage above 250GB required.                                                               |
+| $56/mo         | Network infrastructure costs                                                                                   |
+| +$388/mo min.  | VM instance type - [see "Working out the VM type required" below](#working-out-the-vm-type-required)           |
+| +$1.54/GB      | For each GB of network traffic to/from the instance.                                                           |
```
Comment (bobheadxi) [inline handbook/engineering/distribution/managed/cost_estimation.md:19]: Do we have instructions for how to update these estimates/ how they were calculated?

**Review by bobheadxi — COMMENTED** [review]
nice! left a few questions

File: handbook/engineering/distribution/managed/cost_estimation.md (hunk 2 above)
Comment (dan-mckean) [reply handbook/engineering/distribution/managed/cost_estimation.md:19]: No, updating these estimates is something we talked about needing to do. They're likely out of date.

We also wanted to update the Resourse Calculator as that's probably estimating on the low side of what's really required.

**Review by bobheadxi — APPROVED** [review]

**Review by daxmc99 — APPROVED** [review]
LGTM
Long-term a spreadsheet seems like the best tool here or making the resource calculator automatically pull pricing data.

**Review by davejrt — APPROVED** [review]

**Pull request merged**